### PR TITLE
ocm roles and access ROSA quickstart

### PR DESCRIPTION
**Pull request by aldiazRH**

### CODE REVIEW

File: docs/quickstarts/rosa-grant-ocm-role-access/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+kind: QuickStarts
+name: rosa-edit-cluster-name
```
Comment (florkbr) [inline docs/quickstarts/rosa-grant-ocm-role-access/metadata.yml:2]: @aldiazRH I don't think this is the name we want - should be `rosa-grant-ocm-role-access`

Comment (aldiazRH) [reply docs/quickstarts/rosa-grant-ocm-role-access/metadata.yml]: Thanks for the catch, I've updated it.

**Comment by aldiazRH** [comment]
@florkbr This is now ready for another review and merge. Thank you!

**Comment by florkbr** [comment]
Grype security scan failures are unrelated to this PR - to be followed up on.

**Review by florkbr — APPROVED** [review]

**Pull request merged**